v2/nodejs/session/getSessionInformation.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: getSessionInformation
+title: getSessionInformation
+hide_title: true
+---
+
+## ``getSessionInformation(sessionHandle)``
+
+### Parameters
+- ``sessionHandle``
+  - type : ``string``
+
+### Returns
+- ``Promise<SessionInformation>`` returns all session related information for the given ``sessionHandle``.
+
+### Throws
+- If the `sessionHandle` does not exist
+- If the core version being used is incompatible
```

Comment:
You don't need to say "If the core version being used is incompatible".

Reply:
Done